.gitignore

```diff
@@ -28,5 +28,13 @@ __pycache__
 .pytest_cache
 .python-version
 venv*/
-.idea/
 .mypy_cache/
+
+# jetbrains ide stuff
+*.iml
+.idea/
+
+# vscode ide stuff
+*.code-workspace
+.history
+.vscode
```

Comment:
really all these should be in your gitignore global:

https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files